spec/rubocop/cop/style/double_cop_disable_directive_spec.rb

```diff
@@ -15,6 +15,18 @@ def choose_move(who_to_move) # rubocop:disable Metrics/CyclomaticComplexity
     RUBY
   end
 
+  it 'registers an offense when using `#bad_method` with todo' do
+    expect_offense(<<~RUBY)
+      def choose_move(who_to_move) # rubocop:todo Metrics/CyclomaticComplexity # rubocop:todo Metrics/AbcSize # rubocop:todo Metrics/MethodLength
```

Comment:
How do we handle mixed `rubocop:disable` and `rubocop:todo`? Another example?

Reply:
Based on the current implementation of the cop we're essentially stripping all but the first cop, so that's what it would reduce to.